js/widgets/panel.js

```diff
@@ -106,11 +106,11 @@ return $.widget( "mobile.panel", {
 	},
 
 	_getPanelInner: function() {
-		var panelInner = this.element.find( "." + this.options.classes.panelInner );
+		var panelInner = this.element.find( ".ui-panel-inner" );
 
 		if ( panelInner.length === 0 ) {
 			panelInner = this._safelyWrap( this.element,
-				"<div class='" + this.options.classes.panelInner + "'></div>",
+				"<div class='ui-panel-inner'></div>",
```

Comment:
You have to avoid using ```.wrap()``` and ```.wrapAll()``` if the wrapper has a class added at runtime, because those methods create elements on the fly. So, the elements-created-on-the-fly will not have tracking within the widget factory because they were never passed to ```this._addClass()```. You need to do the wrapping manually. For example,
```JS
var x = $( "<div>" );
this._addClass( x, "ui-panel-inner" );
this.element.before( x );
x.append( this.element );
```